specification/loadtestservice/data-plane/readme.md

```diff
@@ -66,7 +77,7 @@ namespace: com.azure.developer.loadtesting
 partial-update: true
 output-folder: $(azure-sdk-for-java-folder)/sdk/loadtestservice/azure-developer-loadtesting
 service-versions:
-- 2022-06-01-preview
+- 2022-11-01
```

Comment:
Why are there still transforms? It seems like all of these transforms can be incorporated into the spec itself?

Reply:
We have removed the Java section based on previous discussions. The autorest options would now be present in the dedicated SDK repos only